Classes/Events/EventsPayloads/PKAdInfo.swift

```diff
@@ -52,7 +52,11 @@ import Foundation
     @objc public var podIndex: Int
     // The bitrate of the selected creative.
     @objc public var mediaBitrate: Int
-    
+    // The CreativeID for the ad, taken from Google Interactive Media Ads
+    @objc public var creativeID: String
```

Comment:
Please change `creativeID` to `creativeId` to match the style we use elsewhere.

Reply:
Thanks for doing this!